distribution/docs/src/test/java/ddf/docs/DocumentationTest.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.docs;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import org.junit.Test;
+
+public class DocumentationTest {
+
+    String errorMsg = "Unresolved directive";
+
+    @Test
+    public void testDocumentationIncludes() throws IOException {
+        File docPath = new File(new File(this.getClass().getResource("").getFile()).getParentFile(), "docs/html");
```

Comment:
This will most likely _not_ work on Windows. I'd recommend trying something like this instead:

```
Path htmlSubDirectory = Paths.get("docs", "html"); // This should probably be a constant
Paths.get(this.getClass().getResource("").toURI()).resolve(htmlSubDirectory)
```

@Lambeaux, is there a better way to do this that's Windows compatible?